src/main/java/com/automation/school/ad_main/Collection/Address.java

```diff
@@ -0,0 +1,23 @@
+package com.automation.school.ad_main.Collection;
+
+public class Address {
+    public static String country;
+    public static String city;
+    public static String street;
+    //String data[] = {country, city, street};
+    //public String getFullAddress;
```

Comment:
try to remove the commented code if you don't need it

Reply:
Commented code have been deleted